components/AddReference/AddReference.tsx

```diff
@@ -0,0 +1,30 @@
+import styles from "./AddReference.module.css";
+
+function AddReference() {
+  return (
+    <div className={styles.container}>
+      <form>
+        <div>
+          <label>
+            Timecode{" "}
```

Comment:
```suggestion
            Timecode{" "}
```

Do you need this? {" "}